### DISCUSSION TITLE
How do you actually use this right now?

### DISCUSSION THREAD

**Discussion by mybearworld** [NanderTGA/msgroom-orm#182] [Q&A] [answered]
https://msgroom.windows96.net states that "msgroom is down for an indefinite amount of time". But 3.0 was still released - has it been released despite the fact that it's currently not usable, or is there some workaround?

**Comment by NanderTGA** [comment, marked as answer]
Unfortunately, no, since MRCS is down again seemingly.
V3 is just a couple of breaking changes I was going to make anyway.
I mean the `hexColor` type didn't even comply with my usual naming scheme (PascalCase for types, interfaces and classes, CAPITALIZED_SNAKE_CASE for constants that never change like pi and camelCase for anything else) and is inaccurate because it could be any CSS color and it just makes Typescript complain for no reason.
And also the sheesh bot API stuff has been useless for some time now, so yeah I got rid of that too.

When does it make Typescript complain you're asking?
I think you can guess... [Soon](https://github.com/NanderTGA/msgroom-server):tm:

Also, I'm not sure whether I said it somewhere, but we now use semantic-release to automatically publish new releases every Monday that strictly follow semantic versioning. V3 is actually a minor update, and you likely won't notice the difference.

**Reply by mybearworld** [reply]
Ah, that makes sense. Thanks!

> [Soon](https://github.com/NanderTGA/msgroom-server)™

👀

**Comment by NanderTGA** [comment]
[Soon™️](https://nandertga.ddns.net:4096/) is now. Spread the word.
available through https://nandertga.ddns.net:4096 or https://nandertga.ddns.net/msgroom
